tests/helpers.py

```diff
@@ -0,0 +1,44 @@
+"""
+Helper functions for OCS tests
+"""
+import os
+import logging
+import ocs.ocp
+import ocs.defaults as defaults
+from utility import templating
+
+logger = logging.getLogger(__name__)
+
+
+OCP = ocs.ocp.OCP(
+    kind='Service', namespace=defaults.ROOK_CLUSTER_NAMESPACE
+)
+PVC = ocs.ocp.OCP(
+    kind='PersistentVolumeClaim', namespace=defaults.ROOK_CLUSTER_NAMESPACE
+)
+
+TEMP_YAML = os.path.join("templates/ocs-deployment", "temp.yaml")
+TEMPLATES_DIR = "templates/ocs-deployment"
+PROJECT_NAME = 'test-project'
+
+def run_io():
+    """
+    Run IO to a file within the pod
+
+    """
+    OCP.exec_oc_cmd(
+        f"rsh -n openshift-storage ocsci-pod touch /var/lib/www/html/test && "
+        f"dd if=/dev/urandom of=/var/lib/www/html/test bs=1M count=3000 &"
```

Comment:
Can we maybe try to wget some file to mimic a more realistic scenario?
I mean, we don't really sure the urandom is close to real-world IO

Reply:
implemented another mechanism with copying a file in while true running in another thread, while deleting a pvc